src/renderer/components/tree-view/tree-view.tsx

```diff
@@ -0,0 +1,102 @@
+import "./tree-view.scss";
+
+import React, { useEffect, useRef } from "react";
+import { Icon } from "../icon";
+import TreeView from "@material-ui/lab/TreeView";
+import TreeItem from "@material-ui/lab/TreeItem";
+import { cssNames } from "../../utils";
+
+import _ from "lodash";
+import getDeepDash from "deepdash";
+
+const deepDash = getDeepDash(_);
```

Comment:
looking at the docs this isn't needed and we should be able to just do the following:
```suggestion
import { findDeep } from "deepdash";
```

Reply:
I've tryied this, but we'll suddenly hit typescript complaints.
<img width="750" alt="finddeep error" src="https://user-images.githubusercontent.com/9607060/110588880-af34b580-8186-11eb-908b-7945f494ecfa.png">

There's possible to install `deepdash-es` dependency, but it takes also whole new `lodash-es` library as a dependency. And also, there're some test errors occured with `deepdash-es`. So I ended up with current option considering it as necessary evil.

Reply:
oh okay weird. Thanks for the explanation.